src/ServicePulse.Host/build.ps1

```diff
@@ -20,7 +20,9 @@ if ( $? -eq $false ) {
 cd ..
 
 cd $AngularSourceFolder
+$env:NODE_OPTIONS = "--openssl-legacy-provider"
```

Comment:
I may have found a better way of achieving this, that will apply only to angular. I was trying to figure out what this setting actually does, to ensure that it will not have an impact on the final release. I did not figure that out, but I did find [this workaround](https://github.com/nodejs/node/issues/40455#issuecomment-1527619752).

To follow this, add a file named `.npmrc` in the root angular folder. Inside it, add the following:
```js
node-options="--openssl-legacy-provider"
```

Now, all npm commands in this folder will pass that to node. This has the benefit of being in the angular folder, so it will be removed when we move away from angular.

Reply:
I think this works better because it also works for the developer as well and not just for the workflows.  I have updated the PR to reflect the changes.